src/components/SimpleSelect.js

```diff
@@ -44,15 +44,9 @@ class SimpleSelect extends React.Component {
     deprecatePrimaryColor(this.props, 'hoverColor');
     deprecateProp(this.props, 'iconStyles', 'styles', 'simple-select');
     deprecateProp(this.props, 'menuStyles', 'styles', 'simple-select');
-
-    window.addEventListener('keydown', this._handleKeyDown);
-  }
-
-  componentWillUnmount () {
-    window.removeEventListener('keydown', this._handleKeyDown);
   }
 
-  _handleKeyDown = (e) => {
+  _handleKeyUp = (e) => {
```

Comment:
What's the need to change from keyDown to keyUp?

Reply:
Naming convention, since the method now binding to `onKeyUp` listener, it will cause confusions if I left it as `handleKeyDown`.